### PR TITLE
Add snapshot test and bump react-sprucebot

### PR DESCRIPTION
**Pull request by elkwood**
@sprucelabsai/engineers

## Description 
Adds snapshot test for index and bumps react-sprucebot version

## Type
- [ ] Feature
- [ ] Bug
- [x] Tech debt

## Steps to Test or Reproduce
Visit your friendly local skill's index page

### CODE REVIEW

**Comment by elkwood** [comment]
@lkkadiri Does this look okay?

**Comment by elkwood** [comment]
Seems we're missing something in CI or setting up the test runner to generate that `client` config

File: interface/pages/index.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { render } from 'enzyme'
+
+import Index from './index'
+
+test('renders page with styleguide', () => {
+	const tree = render(<Index />)
```
Comment (lkkadiri) [inline interface/pages/index.test.js:7]: 🥇

**Review by lkkadiri — APPROVED** [review]
💯

**Pull request merged**